Update Product_Management.md

shayaanmk opened the pull request:
Added a brief overview of the product management role and linked a couple of articles that would be useful for CSC301 students as beginners, as well as ways to promote practices that lead to becoming successful product managers.

SanjanaGirish commented on Topics/Product_Management.md:
```diff
@@ -1 +1,8 @@
 ## Resources for Product Management
+
+#### Basic Overview
```
I would suggest making the heading more clear. Perhaps 'A beginner's guide to product management' or something more specific to the resources and content you have mentioned.

shayaanmk replied:
Changed the header to be more clear and in-line with the linked articles to provide further clarity.

SanjanaGirish reviewed (changes requested):
Great resources and I think providing beginner resources and articles on the roles and responsibilities of a product manager is a good perspective for CSC301 students. Other students have spoken about what product management entails but creating a beginner's guide on becoming a successful product manager is a unique spin.

I would change the title and try to make the contributions in a separate file to avoid conflict.  Also, make sure to add your contributions with the relevant subtopic in the Readme.

shayaanmk commented:
I have now changed the heading to be more concise and in-line with the linked resources to provide further clarity. I have also updated the main README contents page to include my topic.

SanjanaGirish commented:
Great job, all feedback has been incorporated.
